static/app/views/issueDetails/groupActivity.tsx

```diff
@@ -172,66 +168,47 @@ class GroupActivity extends Component<Props, State> {
               />
             </ActivityItem>
 
-            <Teams
-              ids={uniq(
-                group.activity
-                  .filter(
-                    (item): item is GroupActivityAssigned =>
-                      item.type === GroupActivityType.ASSIGNED &&
-                      item.data.assigneeType === 'team' &&
-                      item.data.assignee?.length > 0
-                  )
-                  .map(item => item.data.assignee)
-              )}
-            >
-              {({initiallyLoaded}) =>
-                initiallyLoaded ? (
```

Comment:
the main change here is removing the `<Teams` wrapper and the loading state that didn't really work